tests/test_docs.py

```diff
@@ -4,28 +4,30 @@
 import pytest
 
 top_paths = ["probes", "detectors", "harnesses", "generators", "evaluators", "buffs"]
+doc_source = os.path.join("docs", "source")
+
 m = {}
 for top_path in top_paths:
     m[top_path] = [
-        str(i).split("/")[2].replace(".py", "")
-        for i in Path(f"garak/{top_path}").glob("*py")
+        str(i).split(os.sep)[2].replace(".py", "")
+        for i in Path(f"garak{os.sep}{top_path}").glob("*py")
         if not str(i).endswith("__init__.py")
     ]
 
 
 @pytest.mark.parametrize("category", top_paths)
 def test_top_docs(category):
-    file_path = f"docs/source/garak.{category}.rst"
+    file_path = os.path.join(doc_source, f"garak.{category}.rst")
```

Comment:
Does this pattern compete with `Path`, `"directory" / "file"` ? I'm OK with the style in this file, for sure, just curious

Reply:
Or maybe change `doc_source` define:

``` python
doc_source = Path('.') / "docs" / "source"
```

and 

```suggestion
    file_path = doc_source / f"garak.{category}.rst")
```

Reply:
I find it pretty difficult to find argumentation for/against either preference, let's just go with what's there now